modules/sdk-coin-sol/src/lib/utils.ts

```diff
@@ -344,7 +344,13 @@ export function validateAddress(address: string, fieldName: string): void {
  * @param network Solana Mainnet or Testnet
  * @returns statics BaseCoin object for the matching token
  */
-export function getSolTokenFromAddress(tokenAddress: string, network: BaseNetwork): Readonly<BaseCoin> | undefined {
+export function getSolTokenFromAddress(
+  tokenAddress: string | undefined,
```

Comment:
how come we made this optional?
can we make required and handle undefine check upstream?